### ISSUE TITLE
Merchant cleanup

### ISSUE THREAD

**Issue by jmg87** [monero-project/monero-site#421]
Hi,
can we get rootbox.host removed as a merchant? Trusting the fact they are added as a merchant in the official docs I purchased a server with them.

Result:

- My moneroj are gone
- No server or anything
- Nobody responding to any of the repeated attempts of communication.

So regardless of if the initial intentions were good or not, at this point it's just a scam or, at best, a black hole to throw your Moneroj in and should be removed.

**Comment by mattcode55** [comment]
>Nobody responding to any of the repeated attempts of communication.

How did you try to contact them? The only contact email I could find anywhere was `kevin@rootbox.host`

**Comment by jmg87** [comment]
I tried via the email (kevin@rootbox.host) , the  internal ticketing system and through reddit. So far no response at all.

It looks like this was a single person project and it was abandoned without notice, but the infrastructure for making the payments is still up. I will let you know if I get any feedback, but the last reddit post from Rootbox was 4 months ago, so I don't expect much.

**Comment by QuickBASIC** [comment]
> Trusting the fact they are added as a merchant in the official docs

Anyone can add a PR to the site and add information. We can't always verify the honesty of merchants. It being listed is not an endorsement by "Monero" (there's not organization just a group of people that decide to contribute).

I think we should consider removing them as long as long as nobody else says they're legitimate.

@RootBox-Kevin We're considering de-listing your site. If you'd like to comment, please do so in this issue.

**Comment by jmg87** [comment]
Hi,

I finally got a response, so I it seems there has been some issues with the service, so, in the end, it does look legitimate. I'll be closing the issue, if it's OK with you all.

**Comment by RootBox-Kevin** [comment]
Hi everyone. Just a quick update here:
jmg87, I now see your email in my inbox at kevin@rootbox.host on the 25th at 12:41 PM. Unfortunately I did not see this email in time. I apologize for this delay. However I noticed at 15:23 on the 25th you created ticket ID#84160. To clarify I did respond at 7:09 on the 26th. We do *try* our absolute best to respond to every ticket sent into support@rootbox.host within 24 hours. I apologize about the hassle here! Moving forward I will try to keep a closer eye on my personal email as well. Rest assured we are legitimate :)
